array.rb

```diff
@@ -0,0 +1,2 @@
+class MyArray < Array
```

Comment:
This is a good start. to get you going i'll give you a few tips

```
def my_inject &block
number_to_return = self.shift
 self.each do |item|
  number_to_return = block.call
 end
end
```


then you can call it like this 
```
arr =  My_Array.new([1,2,3,4])
arr.my_inject{|sum, item| sum + item}
```